style.css

```diff
@@ -0,0 +1,141 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+a,
+a:link,
+a:visited,
+a:focus,
+a:hover,
+a:active {
+  color: #6070ff;
+  cursor: pointer;
+}
+
+:root {
+  --color-white: #fff;
+  --color-light: #7f8cff;
+  --color-light-dark: #6070ff;
+  --color-dark: #172b4d;
+  --color-dark-variant: #344563;
+  --transition: all 400ms ease;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background: #f0f0f0;
+  line-height: 1.7;
+}
+
+/* -------------------------------Styling the Nav Bar-------------------------------- */
+.parent-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header {
+  color: var(--color-light-dark);
+  background: var(--color-white);
+  font-weight: 700;
+  font-size: 18px;
+}
+
+.flex-item-1 {
+  width: 50%;
+  padding: 12px 24px;
+}
+
+.header nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.header nav a {
+  padding-right: 24px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+}
+
+.flex-item-2 {
+  width: 50%;
+}
+
+/* -- -------------------------------Styling the Home Page---------------------------------- -- */
+
+.main-page {
+  background: url('images/background-header.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-bottom-left-radius: 80px;
+}
```

Comment:
- [ ] As it is stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) and suggested by the previous reviewer, kindly use a Flexbox to align the contents of this section. 🤝️ Flexbox gives us a lot of options to align the contents of the page and helps in building responsive pages.

**Project Requirement**
![Screenshot from 2023-01-10 15-19-39](https://user-images.githubusercontent.com/17243325/212061578-60d9d423-88d8-44b5-9de6-f426cf3df4a7.png)

Reply:
@amejid  can you please explain this one part you highlighted? This is only a setting and styling of the background image?

Reply:
@Amen-Musingarimi This page is divided into two parts. The first one is the `header` with the logo and the hamburger in it. The second one is anything remaining outside the `header` tag. You used `display: flex;` on the `header` that's good. According to the requirement the `section` should also utilize `display: flex;`

Reply:
I think your section has a class name `main-page`, that's why we put the suggestion there.

Reply:
So you mean I'm supposed to align both the header and the home page section using flexbox or just the home page section?

Reply:
I don't understand